.pre-commit-config.yaml

```diff
@@ -49,7 +49,7 @@ repos:
       - id: rst
         name: rst
         entry: rst-lint --encoding utf-8
-        files: ^(CHANGES.rst|development.rst|README.rst)$
+        files: ^(README.rst)$
```

Comment:
Don't we want to lint the removed rst-files?

Reply:
> Don't we want to lint the removed rst-files?

We do, and RTD does lint for us. This lint is for files under the project root though and so should be removed since we're asking for it to lint file that no longer exist

Reply:
Got it, thanks! 👍 

Can we still lint on commit somehow? So that you don't have to wait for RTD to do it.

Reply:
I'd need to double check @BeyondEvil ( I think so ) but regardless of whether we do or not this line achieves nothing and should be removed since the files no longer exist.

I made a note to double check linting

Reply:
We also have a tox command to build documentation and it runs in GHA on all PRs, so we definitely don't need to wait on upload to RTD for feedback @BeyondEvil

Reply:
Right, sphinx, so can I assume sphinx will catch linting issues then? @gnikonorov

Reply:
> Right, sphinx, so can I assume sphinx will catch linting issues then? @gnikonorov 
> 
> 

I think so. I made a note to double check this. 

We also can't use this linter since the RST syntax is different ( I remember trying to use it and having it fail in the original doc migration )

Reply:
Opened an issue to investigate linting RTD @BeyondEvil (https://github.com/pytest-dev/pytest-html/issues/428). Can you please review this change? I'd like to get it merged if there are no objections on your end. I'll tackle any linting requirements ( if needed ) in a follow up PR